libpod/healthcheck_linux.go

```diff
@@ -61,7 +62,9 @@ func (c *Container) removeTimer() error {
 		return errors.Wrapf(err, "unable to get systemd connection to remove healthchecks")
 	}
 	defer conn.Close()
-	serviceFile := fmt.Sprintf("%s.timer", c.ID())
-	_, err = conn.StopUnit(serviceFile, "fail", nil)
+	timerFile := fmt.Sprintf("%s.timer", c.ID())
+	if _, err := conn.StopUnit(timerFile, "fail", nil); err != nil && strings.HasSuffix(err.Error(), ".timer not loaded.") {
+		return nil
+	}
 	return err
```

Comment:
This won't work - `err` is only defined in the scope of the if

Reply:
I think you need to do an 
```if err != nil {
  if strings.HasSuffix(err.Error, ...) 
  {
    ...
  } else { 
    return err
  } 
} 
return nil
```

Reply:
right right